docs/getting-started.md

```diff
@@ -34,221 +37,359 @@ The high level data flow is as follows:
 8. The cycle repeats.
 
-### 1.3 What's included?
+### 0.3 What's Included?
 There are two repos relevant to building on The Graph:
 1. [Graph Node](../README.md) (this repo) - A server implementation for indexing, caching and serving queries against data from Ethereum.
 1. [Graph CLI](https://github.com/graphprotocol/graph-cli) - A CLI for building and compiling projects which are deployed to Graph Node.
 
-## 2 Getting started overview
-To deploy a GraphQL endpoint serving blockchain data to your Graph Node we will walk through the following steps:
-
-1. [Create a subgraph project and manifest](#3-defining-your-subgraph)
-1. [Define a GraphQL schema](#31-defining-your-graphql-schema)
-1. [Define your source data](#32-defining-your-source-data)
-1. [Generate types for your mapping with the Graph-CLI](#33-generate-types-for-your-mapping-with-the-graph-cli)
-1. [Write your mappings](#34-write-your-mappings)
-1. [Build and deploy your mappings to IPFS](#41-build-and-deploy-your-mappings-to-ipfs)
-1. [Deploy your subgraph to your local Graph Node](#42-deploy-your-subgraph-to-your-local-graph-node)
-1. [Query your newly deployed GraphQL API](#5-query-your-local-graph-node)
-1. ???
-1. [Profit!](#6-buidl-)
-
-## 3 Defining your Subgraph
-In The Graph, we refer to your project's GraphQL endpoint as a *subgraph*, because once deployed to the decentralized network, it will be just one subset of a global GraphQL endpoint.
+### 0.4 Getting Started Overview
+Below we outline the required steps to build a Subgraph from scratch, that will be serving up queries from a GraphQL endpoint. There are three major steps:
 
-The subgraph is defined as a YAML file called a *subgraph manifest*. See [here](https://github.com/graphprotocol/graph-cli/blob/master/examples/example-event-handler/subgraph.yaml) for an example, or [here](subgraph-manifest.md) for the full subgraph manifest specification. It comprises a schema, source data and mappings which are used to deploy your endpoint.
+1. Define The Subgraph
 
-The subgraph manifest is typically placed into a *subgraph* directory. We'll also add a `package.json` and `tsconfig.json` in the directory, to take advantage of our Javascript-based build toolchain using the Graph CLI.
+    1. [Create a subgraph project and manifest](#3-defining-your-subgraph)
+    
+    2. [Define a GraphQL schema](#31-defining-your-graphql-schema)
+    
+    3. [Define your source data](#32-defining-your-source-data)
+    4. [Generate types for your mapping with the Graph-CLI](#33-generate-types-for-your-mapping-with-the-graph-cli)
+    5. [Write your mappings](#34-write-your-mappings)
+2. Build and Deploy The Subgraph
+    1. [Build and deploy your mappings to IPFS](#41-build-and-deploy-your-mappings-to-ipfs)
+    
+    1. [Deploy your subgraph to your local Graph Node](#42-deploy-your-subgraph-to-your-local-graph-node)
```

Comment:
Can we merge the `build` and `deploy` items into a single one? That's about starting the node and running `yarn deploy`?